web/index.html

```
@@ -3,86 +3,181 @@
+
+    <!-- Mesh Topology -->
+    <div class="panel panel-topology">
+      <div class="panel-hdr">
+        <span class="panel-title">MESH TOPOLOGY</span>
+      </div>
+      <div class="panel-body topo-body">
+        <section class="stagewrap" id="stagewrap" aria-label="Bus topology">
+          <div class="stage" id="stage">
+            <div class="legend" id="legend"></div>
+            <div class="bus"></div>
+            <div class="bus-label">◂◂◂&nbsp;&nbsp;MESH.>&nbsp;&nbsp;▸▸▸</div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)